modules/ROOT/pages/update-workspace.adoc

```diff
@@ -43,36 +55,21 @@ Anypoint Studio includes a new version of the Mule Maven Plugin that can automat
 ----
 |===
 
-Note that this simplification is meant to reduce the number of declarations you need to monitor during development. A fully configured `mule-artifact.json` file is still required by Mule runtime engine when executing the packaged application. You can find the generated `mule-artifact.json` file inside the `target` directory after packaging.
-
-The plugin inspects the metadata declared in `mule-artifact.json`. If there is a value that is not declared, the plugin infers it from the application code before packaging. If other metadata is declared, the plugin uses that information to package the application.
-
-If you update to the latest plugin and want to keep your old descriptor files, you'll need to manually monitor your descriptor metadata.
+This simplification reduces the number of declarations you need to monitor during development. A fully configured `mule-artifact.json` file is still required by Mule runtime engine when executing the packaged application. You can find the generated `mule-artifact.json` file inside the `target` directory after packaging.
 
+The plugin inspects the metadata declared in `mule-artifact.json`. If there is a value that is not declared, the plugin infers it from the application code before packaging. If another metadata is declared, the plugin uses that information to package the application.
```

Comment:
Does "another" mean "other" or "additional"?
And to what does "that information" refer: the first inference or a new inference?
```suggestion
The plugin inspects the metadata declared in `mule-artifact.json`. If a value is not declared, the plugin infers it from the application code before packaging. If another metadata is declared, the plugin uses that information to package the application.
```